### DISCUSSION TITLE
Does doubleClick image option work for mobile?

### DISCUSSION THREAD

**Discussion by wlsquid** [fancyapps/ui#96] [Q&A] [answered]
see https://fancyapps.com/playground/HF 

On mobile/touch devices double click for toggleZoom does not work. Is this intentional? Is there any way around this?

**Comment by fancyapps** [comment]
Hi,

Thank you so much for your feedback, there seems to be a problem and I am working on it.

**Comment by fancyapps** [comment, marked as answer]
This has been fixed in the latest release.

**Reply by wlsquid** [reply]
Cheers
